styletesla.css

```diff
@@ -1,97 +1,774 @@
+    margin: 0 auto;
+  }
+
+  .sidebar__title {
+    padding: 20px 0;
+  }
+
+  .sidebar__discription {
+    padding: 0;
+  }  
+}
+
+@media (max-width: 383px) {
+  .advantage {
+    height: 568px;
+  }
+
+  .sidebar {
+    height: 64%;
+  }
+}
+
+/* 3section */
+.safety {
+  width: 100%;
+  display: grid;
+  grid-template-columns: 20% 80%;
+  grid-template-rows: 100%;
+}
+
+.sidebar {
+  grid-column: 1/3;
+  grid-row: 1/2;
+}
+
+.protect__list {
+  margin-bottom: 100px;
+}
+
+.protect__list_link {
+  display: flex;
+  justify-content: space-between;
+  margin-right: 90px;
+  margin-bottom: 16px;
+  font: 17px/21px montserratbold, sans-serif;
+}
+
+.list_star {
+  height: 22px;
+}
+
+.security {
+  display: flex;
+  align-items: center;
+}
+
+.security__icon {
+  width: 62px;
+  height: 74px;
+  margin-right: 20px;
+}
+
+.star {
+  font: 33px/36px montserratsemibold, sans-serif;
+}
+
+.star_security {
+  font: 25px/30px montserratsemibold, sans-serif;
+}
+
+.safety__img {
+  grid-column: 2/3;
+  grid-row: 1/2;
+  background: url('img/ModelS.png') right no-repeat;
+  background-position-y: 17%;
+  background-size: 70%;
+}
+
+.carinside {
+  padding-top: 175px;
+  padding-left: 50px;
+  width: 90%;
+}
+
+.footer {
+  grid-column: 2/3;
+  display: flex;
+  justify-content: flex-end;
+  align-items: flex-end;
+  }
+
+.footer__main {
+  padding-top: 58px;
+  font: 8px/15px montserratbold, sans-serif;
+}
+
+.footer__webhero {
+  font: 16px/15px montserratbold, sans-serif;
+  color: #ffD22e;
+  cursor: pointer;
+}
+
+.footer__school {
+  font: 11px/15px montserratmedium, sans-serif;
+  color: black;
+  padding-left: 48%;
+}
+
+@media (max-width: 1275px) {
+  .carinside {
+    padding-left: 105px;
+  }
+}
+
+/* планшет */
+@media (max-width: 1165px) {
+  .security,
+  .footer {
+    display: none;
+  }
+
+  .safety {
+    height: 900px;
+    display: grid;
+    grid-template-rows: 57% 43%;
+  }
+
+  .safety__img {
+    grid-column: 1/3;
+    background: url(img/ModelS.png) center no-repeat;
+    background-position-y: 2%;
+    background-size: 55%;
+  }
+
+  .sidebar__safety {
+    height: 100%;
+    grid-row: 2/3;
+  }
+
+  .sidebar__discription_safety {
+    padding-top: 120px;
+    line-height: 22px;
+  }
+
+  .protect {
+    grid-column: 2/3;
+    grid-row: 1/2;
+  }
+
+  .protect__subtitle {
+    padding-left: 0;
+  }
+
+  .protect__list {
+    padding-top: 0;
+  }
+
+  .protect__list_link {
+    margin-right: 155px;
+  }
+
+  .carinside {
+    padding-top: 3%;
+    padding-left: 0;
+    width: 78%;
+    margin: 0 auto;
+  }
+}
+
+@media (max-width: 930px) {
+  .protect__list_link {
+    margin-right: 64px;
+  }
+}
+
+@media (max-width: 790px) {
+  .carinside {
+    padding-top: 11%;
+    width: 88%;
+  }
+
+  .protect__list {
+    padding-left: 0;
+  }
+
+  .list_star {
+    height: 18px;
+  }
+
+  .sidebar__title {
+    font-size: 26px;
+  }
+
+  .safety__img {
+    background-position-y: 14%;
+    background-size: 61%;
+  } 
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/WebHeroSchool/Tesla-12.04/pullRequest?prid=5512124)